src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -347,7 +347,7 @@ try
         // This only occurs if the request handler isn't referenced by the app, which rarely happens if they are targeting the shared framework.
         errorContext.statusCode = 500i16;
         errorContext.subStatusCode = 33i16;
-        errorContext.generalErrorType = "ANCM Request Handler Load Failure";
+        errorContext.generalErrorType = "ASP.NET Core Module Request Handler Load Failure";
```

Comment:
The request handler feels like a nuance best left to error codes/logs to me but I could go either way

```suggestion
        errorContext.generalErrorType = "Failed to load ASP.NET Core runtime";
```

Reply:
So, alternatively:

```suggestion
        errorContext.generalErrorType = "Failed to load ASP.NET Core request handler";
```